frontend/packages/console-app/package.json

```diff
@@ -8,5 +8,8 @@
     "@console/internal": "0.0.0-fixed",
     "@console/plugin-sdk": "0.0.0-fixed",
     "@console/shared": "0.0.0-fixed"
+  },
+  "consolePlugin": {
```

Comment:
This means that Console application itself can contribute extensions, I like that :+1: 

As a follow-up, we can revisit supported extension types and represent related Console application code through `console-app/src/plugin.tsx` (e.g. all the base models).

This also means we'll likely need a way to order plugins before processing their extensions, i.e. `console-app`'s models should be defined _before_ any other models.

I was thinking about e.g. `"order": "before:X"` and `"order": "after:X"` within the `consolePlugin` object, where `X` is the name of the plugin or `after:@console/app` (if not specified, it would fallback to `after:all`).

Reply:
I can see value in having order but not for the current set of use cases we have.
If we can avoid introducing an order altogether for extensions I think it would be great to keep things simple.